plexanisync/logger_adapter.py

```diff
@@ -0,0 +1,8 @@
+import logging
+from typing import Tuple
+
+
+class PrefixLoggerAdapter(logging.LoggerAdapter):
+    """ A logger adapter that adds a prefix to every message """
+    def process(self, msg: str, kwargs: dict) -> Tuple[str, dict]:
+        return (f'[{self.extra["prefix"]}] ' + msg, kwargs)
```

Comment:
In a private repo I have something along the lines of:

```python
import logging
import sys

FORMAT = "%(asctime)s-%(levelname)s -- %(filename)s -- %(message)s"
logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, format=FORMAT)
logger = logging.getLogger()
```

I guess you could have better control over the logging by using the FORMAT option of the logger?